XLS-39d-clawback/README.md

```diff
@@ -0,0 +1,178 @@
+<pre>
+  Title:        <b>Clawback Support</b>
+  Description:  Extending clawback functionality into freeze
+  Revision:     <b>5</b> (2023-5-08)
+<hr>  Author:       <a href="mailto:nikb@bougalis.net">Nikolaos D. Bougalis</a>
+                <a href="mailto:shawnxie@ripple.com">Shawn Xie</a>
+<hr>  Requires:     XLS 39
+<hr>  core_protocol_changes_required:     true
+</pre>
```

Comment:
Should be updated to reflect the xls template. Revision is not used anymore (small standard revisions will be backtracable through git history, major revisions (post XLS) will require an XLS standard number bump)